Tasks/DeployAzureResourceGroup/task.json

```diff
@@ -13,7 +13,7 @@
     "version": {
         "Major": 1,
         "Minor": 0,
-        "Patch": 32
+        "Patch": 35
```

Comment:
why jump from 32 to 35?
are there 2 commits between master and m92 branches?
we have to bump up the master to 36 - if there are no additional changes through out this sprint.